packages/calypso-e2e/src/environment.ts

```diff
@@ -1 +1,11 @@
 export const COOKIES_PATH = process.env.COOKIES_PATH ?? '';
+
+/**
+ * List of locales, used in i18n testing.
+ *
+ * @returns {string[]} Array of lowercase strings.
+ */
+export const LOCALES = (): string[] => {
+	const locales = process.env.I18N_LOCALES ?? '';
```

Comment:
Where is `I18N_LOCALES` set?

Reply:
I've revised it to use LOCALES in a following revision (that I haven't pushed yet). Didn't know the full context of this change so didn't realize that LOCALES was set as a TC param.